src/wallet/ClaimItem.js

```diff
@@ -0,0 +1,32 @@
+import { Fixed8 } from '../utils'
+
+/**
+ * @typedef ClaimItem
+ * @property {Fixed8} claim - Amt of gas claimable
+ * @property {string} txid - Transaction hash of the originaating coin
+ * @property {number} index - Index of coin in the output array
+ * @property {number} value - Amount of NEO involved.
+ * @property {Fixed8} [start] - Starting block. Optional.
+ * @property {Fixed8} [end] - Ending block. Optional.
+ */
+export default class ClaimItem {
```

Comment:
This is considered a breaking change if we do not provide a non-new method to go with it.

Reply:
Reverted ClaimItem back to original format.